content/number-theory/PrecomputeInverses.h

```diff
@@ -8,6 +8,9 @@
  */
 #pragma once
 
-// const ll mod = 1000000007, LIM = 200000; ///include-line
-ll* inv = new ll[LIM] - 1; inv[1] = 1;
-fore(i,2,LIM) inv[i] = mod - (mod / i) * inv[mod % i] % mod;
+constexpr ll mod = 1e9+7, LIM = 2e5;
+array<ll, LIM> inv;
+void initInv() {
+	inv[1] = 1;
+	fore(i,2,LIM) inv[i] = mod - (mod/i) * inv[mod % i] % mod;
```

Comment:
```suggestion
	fore(i,2,LIM) inv[i] = mod - mod / i * inv[mod % i] % mod;
```
Veo que en el original estaban, ¿pero no se podrán sacar esos paréntesis?